pymodbus/factory.py

```diff
@@ -8,6 +8,7 @@
 Regardless of how many functions are added to the lookup, O(1) behavior is
 kept as a result of a pre-computed lookup dictionary.
 """
+
```

Comment:
Why add a blank line ?

Reply:
Oh it's leftover from an `__all__` being here, but then you asking me to remove it.

https://peps.python.org/pep-0008/#module-level-dunder-names has a blank line below the docstring.  I think it looks better